service/test/io/pedestal/interceptor_test.clj

```diff
@@ -196,6 +196,12 @@
                                  (catcher :h)])
                        :leave))))
 
+(deftest t-enque-precondition
```

Comment:
@ordnungswidrig can you add a test with a valid interceptor and drop the call to `execute`? It's not needed to test the precondition.